coala_quickstart/generation/Bears.py

```diff
@@ -90,14 +91,23 @@ def filter_relevant_bears(used_languages,
                 lang_bears, desired_capabilities)
             candidate_bears[lang] = capable_bears
 
-    project_dependency_info = extracted_info.get("ProjectDependencyInfo")
+    lint_task_info = extracted_info.get("LintTaskInfo", [])
```

Comment:
I'm iterating over it, in the next lines, and if None is returned it will throw an error :)